pkg/flags/flagset.go

```diff
@@ -78,3 +82,49 @@ func ParseIgnoredResources() (map[schema.GroupVersionResource]interface{}, error
 	}
 	return gvrs, nil
 }
+
+// ClusterGitTrackMode specifies which mode we're running ClusterGitTracks in
+type ClusterGitTrackMode int
+
+// Enums for ClusterGitTrackMode
+const (
+	CGTMIncludeNamespaced ClusterGitTrackMode = iota
+	CGTMDisabled
+	CGTMExcludeNamespaced
+)
+
+// String implements the flag.Value interface
+func (cgtm ClusterGitTrackMode) String() string {
+	switch cgtm {
+	case CGTMDisabled:
+		return "Disabled"
+	case CGTMIncludeNamespaced:
+		return "IncludeNamespaced"
+	case CGTMExcludeNamespaced:
+		return "ExcludeNamespaced"
+	}
+	panic("unreachable")
+}
+
+// Set implements the flag.Value interface
+func (cgtm *ClusterGitTrackMode) Set(s string) error {
+	lowered := strings.ToLower(s)
```

Comment:
Is this standard practice? 🤔

Reply:
It is a bit DWIM, but I think it'll be useful, especially for `disabled`